src/pages/CreateAccount/CreateAcc.module.scss

```diff
@@ -0,0 +1,92 @@
+.box {
+    padding: 120px 40px;
+    width: 100%;
+    background-color: cornflowerblue;
+  }
+  
+  .createAccountBtn {
```

Comment:
a lot of style duplicates. Create one general style and call it like btn. And then add aditional styles for that main className